conjure-java-core/src/main/java/com/palantir/conjure/java/services/dialogue/DefaultStaticFactoryMethodGenerator.java

```diff
@@ -539,4 +538,23 @@ private static MethodSpec toStringMethod(ClassName className) {
                         StaticFactoryMethodGenerator.ENDPOINT_CHANNEL_FACTORY)
                 .build();
     }
+
+    private enum DeserializerType {
+        STANDARD,
+        EMPTY_BODY,
+        BINARY,
+        OPTIONAL_BINARY;
+    }
```

Comment:
Do we not already have such an enum defined somewhere else? 🤔 Or was it a different repo?